Hearthstone Deck Tracker/Core.cs

```diff
@@ -101,6 +101,11 @@ public static void Initialize()
 					                            "This is either your first time starting HDT or the log.config file has been updated. Please restart Hearthstone, for HDT to work properly.");
 				}
 				LogReaderManager.Start(Game);
+				
+				if(Config.Instance.StartHearthstoneWithHDT && !Game.IsRunning)
```

Comment:
`Game.IsRunning` is currently only updated in `Core.UpdateOverlayAsync` and therefore always false at this point.